common/utils/dates.ts

```diff
@@ -91,3 +91,25 @@ export function getNextWeekendDateRange(date: Date): DateRange {
     end: endOfDay(end),
   };
 }
+
+/** Returns an array containing all the dates between `start` and `end`.
+ *
+ * e.g. getDatesBetween({ start: new Date(2001-01-01), end: new Date(2001-01-04) })
+ *          => [2001-01-01, 2001-01-02, 2001-01-03, 2001-04-01]
+ *
+ */
+export function getDatesBetween({
```

Comment:
This also appears in some of the opening-times code, so it will be reused in `@weco/common` later.